UIComponents/UIComponents/WebServices/Module/News/Controller/NewsViewController.swift

```diff
@@ -51,14 +61,30 @@ class NewsViewController: BaseViewController, Storyboarded {
 extension NewsViewController: UITableViewDelegate, UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        viewModel.article.count
+        article.count
```

Comment:
why we have remove viewmodel articals and use artcals in controller?
View model patten should store or update data and controller just acess it